config/application.rb

```diff
@@ -36,5 +36,14 @@ class Application < Rails::Application
     config.generators do |g|
       g.orm :active_record, primary_key_type: :uuid
     end
+
+    console do
+      require './lib/console_helpers'
+      if defined?(Pry)
+        TOPLEVEL_BINDING.eval('self').extend ConsoleHelpers
+      else
+        Rails::ConsoleMethods.include ConsoleHelpers
+      end
+    end
```

Comment:
Could we avoid the `Pry` complication (and also the implied necessity for a developer to be involved) by making these `rake` tasks (like `rake report:submission_window_progress`)? That would have the added benefit of making this stuff cron-able and with console output, perhaps directed towards Slack for you :smile: